lib/ProMotion/menu/transition.rb

```diff
@@ -0,0 +1,45 @@
+module ProMotion; module Menu
+  module Transition
+
+     VISUAL_STATES = {
+      slide_and_scale:
+        {
+          block: MMDrawerVisualState.slideAndScaleVisualStateBlock,
```

Comment:
Hey @ryanlntn, good catch! The nested hashes are only there because I'd initially intended to include a method that would provide descriptions but then decided against it. I'll modify and update momentarily.